src/odemis/driver/test/technolution_test.py

```diff
@@ -1249,7 +1273,7 @@ def setUp(self):
         self.MPPC.dataContent.value = "empty"
 
         # Change megafield id to prevent testing on existing images/overwriting issues.
-        self.MPPC.filename.value = time.strftime("testing_megafield_id-%Y-%m-%d-%H-%M-%S")
+        self.MPPC.filename.value = time.strftime("test_images/project/project/testing_megafield_id-%Y-%m-%d-%H-%M-%S")
```

Comment:
Is it on purpose that you have a double "project/"?

Reply:
It is fixed now